praw/decorators.py

```diff
@@ -48,7 +48,14 @@ def alias_function(function, class_name):
     """
     @wraps(function)
     def wrapped(self, *args, **kwargs):
-        func_args = inspect.getargspec(function).args
+        if six.PY3 and not hasattr(sys, 'pypy_version_info'):
```

Comment:
Didn't catch that. Yes that would be appropriate.
On Nov 22, 2015 3:38 PM, "13steinj" <notifications@github.com> wrote:

> In praw/decorators.py
> <https://github.com/praw-dev/praw/pull/551#discussion_r45566954>:
>
> > @@ -48,7 +48,14 @@ def alias_function(function, class_name):
> >      """
> >      @wraps(function)
> >      def wrapped(self, *args, **kwargs):
> > -        func_args = inspect.getargspec(function).args
> > +        if six.PY3 and not hasattr(sys, 'pypy_version_info'):
>
> It's used in both alias_function and require_captcha, so internal?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/praw-dev/praw/pull/551/files#r45566954>.
>